test/unit/asset_manager_attachment_metadata_updater_test.rb

```diff
@@ -28,5 +32,11 @@ class AssetManagerAttachmentMetadataUpdaterTest < ActiveSupport::TestCase
       worker.expects(:perform_async).with(attachment_data.id)
       AssetManagerAttachmentMetadataUpdater.update_range(attachment_data.id, attachment_data.id)
     end
+
+    test "does not queue a job on the #{worker} if the file doesn't exist" do
+      FileUtils.rm(attachment_data.reload.file.path)
```

Comment:
Minor: I've used `VirusScanHelpers.erase_test_files` to achieve something similar in other tests, which might be worth considering.

Reply:
We've decided to leave it as it is as we think it makes it clearer that we're deleting the specific file relating to the `AttachmentData`.